parsers/NL.py

```diff
@@ -12,7 +12,7 @@
 
 
 def fetch_production(zone_key='NL', session=None, target_datetime=None,
-                     logger=logging.getLogger(__name__), energieopwek_nl=True):
```

Comment:
do we still get the unknown part from total production, total consumption and exchanges without the flag ?

Reply:
Yes, that happens here:
 https://github.com/tmrowco/electricitymap-contrib/blob/cf34cb7730d6355530c6bda70c397135126cba59/parsers/NL.py#L138-L148